ai-telemetry/base/zookeeper/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
Let's add the namespace here with 0 indent below `kind: Kustomization`: 
```yaml
namespace: zookeeper
```
This will apply the namespace to all resources that this includes.